### PR TITLE
Schema

### PR DESCRIPTION
**Pull request by ACRUNNER**
This is just the schema file with fake data for the three tables.

### CODE REVIEW

**Comment by ACRUNNER** [comment]
[Link to trello board](https://trello.com/c/8kMTX78V/4-find-mock-data-and-add-to-databasemed)

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,1808 @@
+-- ---
+-- Table 'dresses'
+-- Main Dress Item
+-- ---
+DROP TABLE IF EXISTS `rental_time`;
```
Comment (frokrab) [inline schema.sql]: I don't think you need quotes around the table name here

File: schema.sql [hunk 2]
```diff
@@ -0,0 +1,1808 @@
...
+);
+
+-- ---id, warehouse_sent, client_arrived, client_sent, warehouse_arrived, dress_id, size_id
+-- Foreign Keys 
+-- ---
+
+ALTER TABLE `sizes` ADD FOREIGN KEY (product_id) REFERENCES `dresses` (`id`);
+ALTER TABLE `rental_time` ADD FOREIGN KEY (dress_id) REFERENCES `dresses` (`id`);
+ALTER TABLE `rental_time` ADD FOREIGN KEY (size_id) REFERENCES `sizes` (`id`);
+
```
Comment (frokrab) [inline schema.sql]: For all the queries above you don't actually need any of the quotes

File: schema.sql [hunk 3]
```diff
@@ -0,0 +1,1808 @@
...
+--insert data into dresses table
+insert into dresses (id, company_name, name, retail_price, price_4days, average_review_rating, total_reviews, stylist_notes, size_and_fit, product_details, facebook_likes, price_drop_off_percentage) values (1, 'Funk Inc', 'congue risus semper porta', 561, 35, 2.67, 822, 'Integer a nibh. In quis justo. Maecenas rhoncus aliquam lacus.', 'Nunc nisl. Duis bibendum, felis sed interdum venenatis, turpis enim blandit mi, in porttitor pede justo eu massa. Donec dapibus. Duis at velit eu est congue elementum. In hac habitasse platea dictumst. Morbi vestibulum, velit id pretium iaculis, diam erat fermentum justo, nec condimentum neque sapien placerat ante. Nulla justo.', 'Duis bibendum, felis sed interdum venenatis, turpis enim blandit mi, in porttitor pede justo eu massa.', 32, 26);
```
Comment (frokrab) [inline schema.sql]: If you're using auto increment, you don't have to specify the id values for each item-- MySQL will automatically add them to each row. If you want to use auto increment and start counting from somewhere besides 1, you could also add `AUTO_INCREMENT=[number]` as a table option in the create table statement, or add it afterwards with an alter table statement.

**Pull request merged**